apps/docs/src/app/core/component-docs/moment-datetime-adapter/examples/date-picker-moment-adaptor-example.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, LOCALE_ID, ViewChild } from '@angular/core';
+import { DatePickerComponent, DatetimeAdapter } from '@fundamental-ngx/core';
+import moment, { Moment } from 'moment';
+import 'moment/locale/en-gb';
+import 'moment/locale/fr';
+import 'moment/locale/de';
+import 'moment/locale/bg';
+import 'moment/locale/ar';
+
+@Component({
+    selector: 'fd-date-picker-moment-adaptor-example',
+    templateUrl: './date-picker-moment-adaptor-example.component.html',
```

Comment:
The template isn't set inline and it's not shown on the separate tab in examples, when opening StackBlitz template file (`.html`) isn't loaded also.